db_scripts/create_tables.sql

```diff
@@ -318,7 +318,7 @@ CREATE TABLE gridironthrows (
     eventId uuid NOT NULL,
     playerThrowing CHARACTER VARYING(255) NOT NULL,
     playerReceiving CHARACTER VARYING(255) NOT NULL,
-    yard SMALLINT NOT NULL CHECK (yard > 0),
+    yard SMALLINT NOT NULL CHECK,
```

Comment:
do we keep the `CHECK`?

Reply:
no that's clearly a mistake from sloppy editing.

Reply:
fixed.